README.md

```diff
@@ -1 +1,204 @@
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Display_Hello_Microverse]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- Live Demo Link(https://yacoubou-seidou.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+Installation of Git
+Installation of NPM
+A working browser
+Internet connection
+A code editor like Visual Studio Code
+Installation of Live Server plugging in Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+cd my-folder
+git clone git@github.com:myaccount/my-project.git
+
+### Install
+
+Install this project with:
+
+cd my-project
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+Open with live server
+Open in Browser
+
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **YacoubouSeidou**
+
+- GitHub: @githubhandle(https://github.com/yacoubou-seidou)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Display_hello_Microverse]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project give a start to the repository and follow my github accouunt
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all my coding partners from Microverse
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[How_many_times_before_finalizing_this_project]**
+
+  - More_then_3_hours
+
+- **How*was_the_difficulty*?**
+
+  - It_was_hard_to_follow_all_the_project_needs
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please add a LICENSE to your project. A LICENSE lets people know what/ how they can use your project. Follow this [resource](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to add a licence in your repo 😄

Reply:
@wayungi, For LICENSE should I make a new branch?